src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -355,6 +355,48 @@
                             </xs:annotation>
                         </xs:attribute>
 
+                        <xs:attribute name="logRollingStrategy" default="size" use="optional" >
+                            <xs:annotation>
+                                <xs:documentation>
+                                    The strategy to use for rolling over the log file. Default is size.
+                                </xs:documentation>
+                            </xs:annotation>
+                            <xs:simpleType>
+                                <xs:restriction base="xs:string">
+                                    <xs:enumeration value="size">
+                                        <xs:annotation>
+                                            <xs:documentation>
+                                                The log file will be rolled over when it reaches the size specified in maxLogFileSizeBytes.
```

Comment:
I know it's common practice to configure this as bytes, but I doubt most users need that kind of granularity, and it's easy to miscount the number of zeros when editing the value. What do you think about making the rollover size be in MB?

Reply:
Seems reasonable.